docs/content/en/docs/tasks/packages/troubleshoot.md

```diff
@@ -186,9 +191,13 @@ In the example above, the secret does not exist which may be that the management
 
 This also may happen if the management cluster cannot communicate with the workload cluster or the workload cluster was deleted, although the detail would be different.
 
-### Error: Error from server (NotFound): packagebundlecontrollers.packages.eks.amazonaws.com "clusterName" not found
+### Error: the server doesn't have a resource type "packages"
+
+All packages are remotely managed by the management cluster, and packages, packagebundles packagebundlecontrollers resources are all deployed on the management cluster. Please make sure the kubeconfig is pointing to the management cluster that was used to create the workload cluster while interacting with package related resources.
```

Comment:
```suggestion
All packages are remotely managed by the management cluster, and packages, packagebundles, and packagebundlecontrollers resources are all deployed on the management cluster. Please make sure the kubeconfig is pointing to the management cluster that was used to create the workload cluster while interacting with package-related resources.
```
Please check that my change reflects what you meant to say.

Reply:
Thank you for your review! I addressed both comments.